06-building-decays.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: page
+title: First Steps in LHCb
+subtitle: Building your own decay
+minutes: 15
+---
+
+> ## Learning Objectives {.objectives}
+>
+> * Learn the concepts behind the LHCb selection framework
+> * Build a decay chain 
+
+In order to perform most physics analyses we need to build a *decay chain* with reconstructed (or MC) particles that represents the physics process we want to study.
+In LHCb, this decay chain can be built through `LHCb::Particle` (`LHCb::MCParticle`) objects that represent individual particles and contain links to their children, also represented by the same type of object.
+
+We'll learn all the concepts involved by running through a full example:
+using the DST file we downloaded in the [Downloading a file from the Grid](05-files-from-grid.html) lesson, we will build our own $D^*\rightarrow D^0(\rightarrow K \pi) \pi$ decay chain from scratch.
+Get your [LoKi skills](https://lhcb.github.io/first-analysis-steps/06-loki-functors.html) ready and let's start.
+
+The typical approach is to build the decay from the bottom up. Therefore, we need to
+
+  1. Get input pions and kaons and filter them according to our physics needs.
+  2. Combine a pion and a kaon to build a D0, filtering in the necessary physics quantities.
```

Comment:
I don't like the english "filtering in the necessary physics". How about "and applying selection cuts to the D0."